virt/fusion_access_SAN/install-configure-fusion-access-san.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="install-configure-fusion-access-san"]
+= Installing and configuring {IBMFusionFirst}
+include::_attributes/common-attributes.adoc[]
+:context: install-configure-fusion-access-san
+
+toc::[]
+
+To use Red Hat {VirtProductName} with {IBMFusionFirst}, you must first install the {FusionSAN} Operator.
+
+Then you must create a Kubernetes pull secret and create the FusionAccess custom resource (CR).
+
+Finally, follow the Red Hat {product-title} web console wizard to configure the storage cluster, local disk, and file systems.
+
+include::modules/virt-installing-fusion-access-operator.adoc[leveloffset=+1]
+
+include::modules/virt-creating-pull-secret-fusion-san.adoc[leveloffset=+1]
+
+include::modules/virt-creating-fusionaccess-cr.adoc[leveloffset=+1]
+
+include::modules/virt-creating-storage-cluster-fusion-access-san.adoc[leveloffset=+1]
+
+include::modules/virt-creating-filesystem-fusion-access-san.adoc[leveloffset=+1]
+
+== Next steps
```

Comment:
I wonder if this heading needs an id. Please double-check.

Reply:
It does, thanks for catching it.